util/issue_commands.go

```diff
@@ -99,3 +100,19 @@ func SendUpdateRedeemClaim(goCtx context.Context, beneficiary string, id uint64,
 	loggingContext := "redeem claim"
 	buildSignBroadcastTx(goCtx, loggingContext, sendingValidatorAddress, msg)
 }
+
+func SendPLMNTTokens(goCtx context.Context, beneficiary sdk.AccAddress, amount uint64, denominator string) {
+	sendingValidatorAddress := config.GetConfig().ValidatorAddress
+
+	coin := sdk.NewCoin(denominator, sdk.NewIntFromUint64(amount))
+	coins := sdk.NewCoins(coin)
+	orgAddr, err := sdk.AccAddressFromBech32(sendingValidatorAddress)
```

Comment:
If it can be assured that `config.GetConfig().ValidatorAddress` is validated during the startup procedure this can be replaced with `sdk.MustAccAddressFromBech32(sendingValidatorAddress)` thus removing the need for the error check

Reply:
I changed it. We've been using the Must version in an optimistic way as we never verify the configuration. I will create a ticket so that we read it from the keystore instead of having this configured. it's in any case not so beautiful.